dimagi/pages/static/style/pages/sectors/agricultural.scss

```diff
@@ -0,0 +1,123 @@
+@import 'app/variables/type';
+
+#page-intro {
+  $_base-height: 190px;
+  $_height-by-size: (
+    small: 280px,
+    medium: 280px,
+    large: 280px,
+    extra-large: 340px,
+  );
+  @include make-bg(hero-data-image, '../../../pages/sectors/agriculture-extension-programs/intro/hero', $_base-height, $_height-by-size);
+}
+
+.news-images .image.blog1 {
+  @include background-cover('../../../pages/sectors/agriculture-extension-programs/news/AgExtension_Blog1.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.news-images .image.blog2 {
+  @include background-cover('../../../pages/sectors/agriculture-extension-programs/news/AgExtension_Blog2.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.news-images .image.blog3 {
+  @include background-cover('../../../pages/sectors/agriculture-extension-programs/news/AgExtension_Blog3.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.programs-images .image.featured-project1 {
+  @include background-cover('../../../pages/sectors/agriculture-extension-programs/programs/AgExtension_FeaturedProject1.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.programs-images .image.featured-project2 {
+  @include background-cover('../../../pages/sectors/agriculture-extension-programs/programs/AgExtension_FeaturedProject2.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.programs-images .image.featured-project3 {
+  @include background-cover('../../../pages/sectors/agriculture-extension-programs/programs/AgExtension_FeaturedProject3.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+
+.modal-popup {
+  width: 309px !important;
+}
+
+.part3 {
+  $_base-height: 550px;
+  $_height-by-size: (
+    small: 620px,
+    medium: 385px,
+    large: 360px,
+    extra-large: 360px,
+  );
+  @include make-bg(epic-cover, '../../../pages/sectors/agriculture-extension-programs/part3/hero', $_base-height, $_height-by-size);
+}
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.commcare-text {
+  $_base-height: 290px;
+  $_height-by-size: (
+    small: 290px,
+    medium: 290px,
+    large: 290px,
+    extra-large: 290px,
+  );
+  @include make-bg(phone-image, '../../../pages/sectors/agriculture-extension-programs/news/Phone', $_base-height, $_height-by-size, '.png');
+
+  @media (min-width: $width-desktop) {
+    .off-grid {
+      top: 10%;
+      left: 0;
+      position: absolute;
+      right: auto;
+      width: 120%;
+      z-index: 1;
+    }
+
+    .content {
+      position: relative;
+      z-index: 2;
+    }
+  }
+}
+
+.part3 .image .bg-loader{
+  background-color: transparent;
+}
+
+.mb-10 {
+  margin-bottom: 10px;
+}
+
+.touch .sub-nav a:not(.button).primary span, .no-touch.sub-nav-fixed-top .sub-nav a.primary span {
+  background-color:#47b700;
```

Comment:
hex colors should never be referenced directly. please, please, please use the variables in `colors.scss`.